chanreg.md

```diff
@@ -122,7 +122,9 @@ Links to places we can find out more about your project:
 # About you and your staff
 Your NickServ account:
 Your relationship to / position in the project:
-NickServ of group contacts and if they're primary/secondary, hidden/public:
+NickServ accounts of group contacts:
+(and if they're primary/secondary, hidden/public)
+(please remember to include yourself if you would like to be a contact)
```

Comment:
Not sure if I think they need to, I believe this is inferred from "your NickServ account" above.
Also not sure if I think it needs to be separate parenthesis, could be one with a comma imo

Reply:
They do need to at least indicate whether they want to be primary/secondary, hidden/public. The only sane way to do that is to put themselves where it asks for those details, IMO.

Reply:
Since we aren't doing primary/secondary stuff anymore, I think it should be something like:
```
NickServ accounts of group contacts and if they should be hidden or public:
(Please remember to include yourself if you would like to be a group contact)
```